libs/core-ui/src/lib/util/getDependencyChartOptions.ts

```diff
@@ -57,7 +60,8 @@ export function getDependencyChartOptions(
       }
     ],
     xAxis: {
-      categories: xLabels
+      categories: xLabels,
+      type: logarithmicScaling ? AxisTypes.Logarithmic : undefined
```

Comment:
just curious, there are two types of axis. If it isn't logarithmic it would be Liner. Why undefined?

Reply:
The reason to set it to undefined is, from Highcharts wiki: `The type of axis. Can be one of linear, logarithmic, datetime, category or treegrid. Defaults to treegrid for Gantt charts, linear for other chart types.` In case we set `treegrid` for `Gantt charts`, I would just keep it to `undefined`, so it can select by default value according to chart type.